### PR TITLE
Dbg week updates

### PR DESCRIPTION
**Pull request by samiurkh1n**
Hey y'all. Content for debugger week.

Three changes in this PR:

1. updates syllabus with summary and release times
2. adds pdfs of hw and advanced hw
3. uploads slides for the week

Could not get docker to work so couldn't generate pdf using the docker command. Hoping someone else could do that in a follow up?

### CODE REVIEW

**Comment by tarunsk** [comment]
Yeah, I'll generate the pdf for you. Docker is messy so I set up the system locally

File: static/w18/hw/c4cs-wk11-homework.tex [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+    - check if N is prime
+      * if no prime n between 1..sqrt(N) divides N, then prime
+    - save whether N is prime for future loops to use
+    - if N is prime, print N
+\end{lstlisting}
+
+\subsection{Debugging with \texttt{gdb}}
+
+\emph{You may find it helpful to consult the
+  \href{https://c4cs.github.io/lectures/f17/week11}{gdb lecture notes}
```
Comment (tarunsk) [inline static/w18/hw/c4cs-wk11-homework.tex]: I think this link needs to be updated to point to this semester's lecture slides

File: static/w18/advanced/c4cs-wk11-advanced.tex [hunk 2]
```diff
@@ -0,0 +1,419 @@
...
+
+\begin{document}
+
+\fancyfoot[L]{\color{gray} C4CS -- W'18}
+\fancyfoot[R]{\color{gray} Revision 1.0}
+\fancyfoot[C]{\color{gray} \thepage~/~\pageref*{LastPage}}
+\pagestyle{fancyplain}
+
+\title{\textbf{Advanced Homework 11\\Debugging}}
+\author{\textbf{\color{red}{Due: Wednesday, March 27th, 11:59PM (Hard Deadline)}}}
```
Comment (AmritHariharan) [inline static/w18/advanced/c4cs-wk11-advanced.tex]: should be the 28th right? (same for the other file)

File: static/w18/hw/c4cs-wk11-homework.tex [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+\fancyhead[L]{}
+\fancyhead[R]{}
+
+\fancyfoot[L]{\color{gray} C4CS -- W'18}
+\fancyfoot[R]{\color{gray} Revision 1.0}
+\fancyfoot[C]{\color{gray} \thepage~/~\pageref*{LastPage}}
+\pagestyle{fancyplain}
+
+\title{\textbf{Homework 11\\Debugging}}
+\author{\textbf{\color{red}{Due: Wednesday, March 27th, 11:59PM (Hard Deadline)}}}
```
Comment (tarunsk) [inline static/w18/hw/c4cs-wk11-homework.tex]: Due date should be 28th

File: static/w18/advanced/c4cs-wk11-advanced.tex (hunk 2 above)
Comment (samiurkh1n) [reply static/w18/advanced/c4cs-wk11-advanced.tex]: Fixed with commit 83d1ef0

File: static/w18/hw/c4cs-wk11-homework.tex (hunk 1 above)
Comment (samiurkh1n) [reply static/w18/hw/c4cs-wk11-homework.tex]: Fixed with commit 83d1ef0

File: static/w18/hw/c4cs-wk11-homework.tex (hunk 3 above)
Comment (samiurkh1n) [reply static/w18/hw/c4cs-wk11-homework.tex]: Fixed with commit 83d1ef0

**Pull request merged**